monitor.py

```diff
@@ -240,13 +241,19 @@ def cluster_deploy_status(spy_link):
         except json.JSONDecodeError as e:
             return 'ERROR'
     else:
+        pattern=r"(\d+\.\d+)"
+        match=re.search(pattern,spy_link[8:])
+        version=float(match.group(1))
         job_log_url = PROW_VIEW_URL + spy_link[8:] + '/artifacts/' + job_type + '/ipi-install-' + job_platform +'-install/finished.json'
         if "sno" in spy_link:
             job_log_url = PROW_VIEW_URL + spy_link[8:] + '/artifacts/' + job_type + '/upi-install-powervs-sno/finished.json'
         
         # temporary fix for getting cluster deployment status of jobs where clusters are deployed by upi
         if "4.16-ocp-e2e-ovn-remote-libvirt-ppc64le" in spy_link:
```

Comment:
Do we need this line if we are checking for version > 4.16? We can remove this check right ?

Reply:
No, we can modify match pattern for upgrade jobs . 
4.15-4.16 upgrade . here version will be 4.15 acc to our scripts. but it is 4.16